src/style.css

```diff
@@ -1 +1,87 @@
 /* add styles here */
+@font-face {
+  font-family:Avenir;
+  font-style: normal;
```

Comment:
Default value, so you can just remove it everywher
```suggestion
```

Reply:
isn't fixed